Add cartocss file

csobier opened the pull request:
@ohasselblad Can you view this branch in grunt serve and review the "Customizing Maps with CartoCSS  / *CartoCSS Properties for Torque Style Maps* section?  There are some reviewer questions that I need answered before I can publish this branch.

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+
+<span class='reviewer'>reviewer: IS THIS CORRECT, OR DO THESE PROPERTIES ONLY APPLY TO TORQUE MAPS? It looks like I can see all these properties (behind the scenes in the CartoCSS Editor) for all Torque style maps</span>
+
+| | [torque-frame-count](#torque-frame-count-numbers) | [torque-animation-duration](#torque-animation-duration-numbers) | [torque-time-attribute](#torque-time-attribute-string)
```
All of the properties should begin with a hyphen, like this: '-torque-frame-count'

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+| | [torque-aggregation-function](#torque-aggregation-function-keyword) | [torque-resolution](#torque-resolution-float) | [torque-data-aggregation](#torque-data-aggregation-keyword)
+
+
+##### torque-frame-count `numbers`
```
I'd just put 'number' instead of the plural

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+
+------------ | -------------
+Description | Specifies the number of steps in your torque animation.
+Sample CartoCSS Code | `torque-frame-count: 512;`
```
Again, use a hyphen to begin the torque rule: '-torque-frame-count: 512;'

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+------------ | -------------
+Description | Specifies the number of steps in your torque animation.
+Sample CartoCSS Code | `torque-frame-count: 512;`
+Default Value | undefined
```
Is it really undefined? Or depends on input data?

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+Description | Specifies the length of time for your animation, in seconds.
+Sample CartoCSS Code | `torque-animation-duration: 30;`
+Default Value | undefined
+Available Values | See [numbers](#numbers).
```
This can be a real number (decimal) while frame count has to be an integer?

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+Sample CartoCSS Code | `torque-animation-duration: 30;`
+Default Value | undefined
+Available Values | See [numbers](#numbers).
+| <span class='reviewer'>reviewer: IS THERE A SPECIFIED RANGE?</span>
```
For specified range, I think it can be from 0 - pretty large, as in billions of seconds. @fdansv knows :)

fdansv replied:
The range is any positive integer (although it will work with decimals), with no documented maximum.

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+------------ | -------------
+Description | Since Torque maps renders data in clusters, this property defines how values are displayed in each cluster of the map. For example, you can define: a maximum value, a count, or the total number of values in each cluster.
+Sample CartoCSS Code | `torque-aggregation-function: "count (cartodb_id)";`
+Default Value | undefined
```
default is `count(cartodb_id)`

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+Sample CartoCSS Code | `torque-aggregation-function: "count (cartodb_id)";`
+Default Value | undefined
+Available Values | `count(column_name), max(column_name), sum(column_name)`
+| <span class='reviewer'>reviewer: I WAS TOLD THAT THERE ARE ACTUALLY MORE AGGREGRATION FUNCTIONS THAT ARE SUPPORTED?  PLEASE ADVISE.</span>
```
Yes, see a large list here: http://www.postgresql.org/docs/9.3/static/functions-aggregate.html

Functions can also combinations of functions and operations, like `log(1 + max(column_name))`

ohasselblad commented on _includes/cartocss_torque.html:
```diff
@@ -0,0 +1,128 @@
+Sample CartoCSS Code | `torque-resolution: 2;`
+Default Value | undefined
+Available Values | See [float](#float).
+| **Note:** Defining a larger number value applies more grids to your data.
```
I would put 'applies a larger grid' instead of 'applies more grids'

ohasselblad replied:
It should also be noted that resolution should be in powers of 2: 1, 2, 4, 8, ... up to 256

csobier commented:
@rochoa, Congratulations, AXH has volunteered you to be the final reviewer of this content.  If everything appears correctly, please merge this branch.  (The "Customizing Maps with CartoCSS" section needs your sign-off).

**Note:** We are adding CartoCSS content in phases. This phase contains all the CartoCSS properties and values.  Subsequent doc updates will include examples and references for each parameter. (Please note: I have a list of outstanding questions from other reviewers that I will send to you offline that can enhance this content). 

*We would like to merge this by end of day Tuesday (for the Cabra del Santo Cristo release). Please notify me if there are any stoppers that need to be edited in this branch before we can release.

Thank you!

andrewxhill commented:
@rochoa can you give this a quick read through in the morning for glaring errors in what is possible on cartodb version of css?

csobier commented:
@axh, I removed the lone image and citation to the Mapbox example in the comp-op section, and added a note that we will be updating the doc in a future release.

@rochoa We missed our targeted release for yesterday's milestone.  Please let me know what I can to help you get through a quick review and sign-off? I'm happy to help.

fdansv commented:
Hey, I'm reviewing this since Raúl is busy dealing with the tiler's release. I'm taking a look now, will update in a bit.

fdansv commented:
@andrewxhill @csobier I can't spot anything obviously erroneous, but understand it's a bit of an overwhelming amount of documentation :sweat_smile: I say let's merge it. Since a few of us will use the reference quite often it'll be simple to keep an eye out for it.

:ship: :it:

csobier commented:
Thanks @fdansv , I will merge.  (We will have several iterations of doc updates on this topic.  The next [milestone](https://github.com/CartoDB/docs/milestones/Horcajo%20de%20Santiago) already has several issues assigned.)

Note: There is also a "Stopper" milestone should anyone find a huge error once this is live- please report.  Otherwise, we will continue to enhance this content.
